tensorflow_lite_support/ios/task/vision/utils/sources/GMLImageUtils.m

```diff
@@ -137,10 +143,10 @@ + (nullable UInt8 *)populateRGBBufferFromSourceRGBABuffer:(UInt8 *)buffer
   if (!buffer_to_return) {
     return nil;
   }
-  for (int row = 0; row < width; row++) {
-    for (int col = 0; col < height; col++) {
-      long offset = sourceChannelCount * (col * width + row);
-      long rgbOffset = destChannelCount * (col * width + row);
+  for (int col = 0; col < width; col++) {
```

Comment:
@priankakariatyml Was there a bug that you just uncovered? Could you give me more details about what when wrong? 

Does the tests in ImageClassifier and ObjectDetector still pass after this change?

Reply:
I just renamed `row` to `col` and `col` to `row` here. The logic is the same. This was just a variable name change and doesn't fix anything. Tests run fine.  My fix was [here](https://github.com/khanhlvg/tflite-support/blob/ios-image-segmentation/tensorflow_lite_support/ios/task/vision/utils/sources/GMLImageUtils.m#L112) for incorrect reading of segmentation input image pixel values. You can see my explanation in comments above this line.

Reply:
I see. Thanks for the clarification.